nixos/modules/security/wrappers/wrapper.c

```diff
@@ -151,9 +154,55 @@ static int make_caps_ambient(const char *self_path) {
     return 0;
 }
 
+// These are environment variable aliases for glibc tunables.
+// This list shouldn't grow further, since this is a legacy mechanism.
+// Any future tunables are expected to only be accessible through GLIBC_TUNABLES.
+//
+// They are not included in the glibc-provided UNSECURE_ENVVARS list,
+// since any SUID executable ignores them. This wrapper also serves
+// executables that are merely granted ambient capabilities, rather than
+// being SUID, and hence don't run in secure mode. We'd like them to
+// defend those in depth as well, so we clear these explicitly.
+//
+// Except for MALLOC_CHECK_ (which is marked SXID_ERASE), these are all
+// marked SXID_IGNORE (ignored in secure mode), so even the glibc version
+// of this wrapper would leave them intact.
+#define UNSECURE_ENVVARS_TUNABLES \
```

Comment:
What is our process to review the list of new environment variables in future?
Since we no longer use glibc, this is now a manual task now.

Reply:
I think this was already mentioned somewhere in this (long) PR thread, but: this hardcoded list is not expected to grow, since glibc has switched to controlling these things through `GLIBC_TUNABLES` instead. And other environment variables are grabbed directly from the glibc source (UNSECURE_ENVVARS).